sqlmesh/dbt/manifest.py

```diff
@@ -260,6 +270,18 @@ def _load_profile(self) -> Profile:
             target_override=self.target.name,
         )
 
+    def _macro_dependencies(
```

Comment:
Why do we need this? Doesn't the top-level node capture all nested macro dependencies already?

Reply:
Ah, I guess because we need to capture additional sources and refs now